QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002108	$The record which I can use to find the whereabouts of all the Goddesses… {nl}If you could obtain that secret information, we might be able to predict the Goddesses' future locations by analyzing what they did in the past. {nl}Because of that, your future plans may be disturbed or put at risk. {nl}
+QUEST_20150714_002109	$This is more dangerous because, if the demons find out the truth, their greed towards the library will grow, {nl}and if that greed is fulfilled, the world will be swallowed by evil. That's why…
+QUEST_20150714_002110	$(The rest is blurred and unreadable.)
+QUEST_20150714_002111	$The Memo of Disciple Hones
+QUEST_20150714_002112	$If you want to find the master's research, you have to find three spatial magic stones and activate the fusion device.
+QUEST_20150714_002113	$The spatial magic stone of the contract is at the reception room. {nl}The monsters probably have the fragments. {nl}Collect all the fragments of the magical stones and go to the preserved magic field in the Grave of the Saints to restore them.
+QUEST_20150714_002114	$The spatial magic stones of sacrifices are with the Templeshooter Witches. {nl}The guiding stone of wisdom will help you to find the hidden Templeshooter.
+QUEST_20150714_002115	$The spatial magic stones of the promise can be found in Siaura Mine. {nl}Use the essence of the spirit to the monsters first to collect their condensed resentment. {nl}With that, destroy the stones in Siaura Mine. {nl}Black chasers will pursue from behind, avoid them and go to the end of the mine to find the spatial magic stones of the promise.
+QUEST_20150714_002116	$When you find all three spatial magic stones, go to the fusion device again and place all the magic stones there. {nl}The path to the master's research will then open at the Raudi Building.
+QUEST_20150714_002117	$Spirit
+QUEST_20150714_002118	$Please… save us from this horrible contract… {nl}Remove us from this… contract of declaration…
+QUEST_20150714_002119	$The contract of declaration… {nl}It expires as we speak… I have no regrets…
+QUEST_20150714_002120	$Eternal imprisonment… from the restraints of the contract… {nl}Break… the pot of silence…
+QUEST_20150714_002121	$The pot of silence is at Memum Corridor… {nl}I'm so… tired…
+QUEST_20150714_002122	$Where to start…
+QUEST_20150714_002123	$I was a disciple of Agailla Flurry when I was alive. {nl}I didn't have any magical abilities so I couldn't use magic.
+QUEST_20150714_002124	$That's why I survived Helgasercle's wrath. {nl}He must've thought he didn't have to kill me since I wasn't able to use magic.
+QUEST_20150714_002125	$I am confident in my knowledge of terms and theories more than the other disciples. {nl}That's why I studied magic. And while I was studying, I got to know the magic stone of interaction.
```

Comment:
> I am confident in my knowledge of terms and theories more than the other disciples. 

"I am confident I am better when it comes to theory than other disciples." The original was "in terms of theory", so it can't be "in terms and theories".

> And while I was studying, I got to know the magic stone of interaction.

During my study, I learned about the magic stone of interaction.